versions/1/changelog.yml

```diff
@@ -7,6 +7,9 @@
   - description: Prepare for next version
     type: enhancement
     link: https://github.com/elastic/package-spec/pull/364
+  - description: Refactored benchmark field in kibana/csp-rule-template asset 
+    type: enhancement
+    link: https://github.com/elastic/package-spec/pull/365
```

Comment:
This PR is only changing a sample test package, there is no user-facing change. Do we need a changelog entry for that?
```suggestion
```